<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>1</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="labelOp1">
        <property name="text">
         <string>Wert 1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="inputOp1"/>
      </item>
      <item>
       <widget class="QLabel" name="labelOp2">
        <property name="text">
         <string>Wert 2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="inputOp2"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="operationRadioGroup" stretch="1,1,1,1">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <widget class="QRadioButton" name="radioAdd">
          <property name="text">
           <string>Addieren</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioMul">
          <property name="text">
           <string>Multiplizieren</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioSub">
          <property name="text">
           <string>Subtrahieren</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioDiv">
          <property name="text">
           <string>Dividieren</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="labelResult">
        <property name="text">
         <string>Noch kein Ergebnis</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="actionGroup">
        <item>
         <widget class="QPushButton" name="actionCalculate">
          <property name="text">
           <string>Berechnen</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="actionSwapOperands">
          <property name="text">
           <string>Operanden tauschen</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="actionReset">
          <property name="text">
           <string>Zur√ºcksetzen</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="actionQuit">
          <property name="text">
           <string>Beenden</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
